doc/ApiDocGeneration/docindex.yml

```diff
@@ -0,0 +1,47 @@
+resources:
```

Comment:
Can you add an entry to https://github.com/Azure/azure-sdk-for-net/blob/master/doc/README.md with a brief explanation for that this directory is for?

Reply:
Added